### ISSUE TITLE
TypeError: this.writeContracts is not a function

### ISSUE THREAD

**Issue by rocky** [ConsenSys/truffle-security#56]
```
/src/external-vcs/github/consensys/devcon4-playground/exercise3/contracts/Etherbank.sol:17:22: Warning: Unused local variable.
      (bool success, bytes memory data) = msg.sender.call.value(amount)("");
                     ^---------------^

TypeError: this.writeContracts is not a function
    at compilers.map (/src/external-vcs/github/consensys/truffle-security/node_modules/truffle-workflow-compile/index.js:85:20)
```

**Comment by rocky** [comment]
I have a feeling this is a truffle version mismatch thing.

**Comment by rocky** [comment]
Fixed via f7108199e015ef7486be60ff139ee33d1754ef65
